.github/workflows/ci.yml

```diff
@@ -63,7 +63,15 @@ jobs:
           override: true
 
       - name: Install scopelint
-        run: cargo install scopelint
+        uses: engineerd/configurator@v0.0.8
```

Comment:
_(arbitrary comment location to start a thread)_

I don't see the `coverage` step added here (it was on a branch until now, just merged). Maybe you left it off intentionally which is fine, just calling that out in case you want to add it

Reply:
Ahh yeah I pulled the updates from `main` and it wasn't added yet. Now include in 29b3bedc50644c79b9f947fe63322c7a78faefd3